backend/api/Areas/Keycloak/Mapping/User/UserMap.cs

```diff
@@ -23,6 +26,15 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.Roles, src => src.Roles)
                 .Inherits<Entity.BaseEntity, Api.Models.BaseModel>();
 
+            config.NewConfig<Entity.User, KModel.UserModel>()
+                .Map(dest => dest.Id, src => src.Id)
+                .Map(dest => dest.Username, src => src.Username)
```

Comment:
Here we map to "Username" but in the other mapper you mapped to another model with "UserName" (upper-case N)
I'd prefer some consistent naming across all of out models if possible. Would you be open to rename the other model to match what's already in place?

Reply:
It should be `Username`, as it is not the User's Name, but their unique identifier Username